modules/dxp/apps/segments/segments-asah-connector-test/src/testIntegration/java/com/liferay/segments/asah/connector/internal/messaging/test/IndividualSegmentsCheckerTest.java

```diff
@@ -0,0 +1,200 @@
+ * distribution rights of the Software.
+ *
+ *
+ *
+ */
+
+package com.liferay.segments.asah.connector.internal.messaging.test;
+
+import com.liferay.analytics.settings.configuration.AnalyticsConfiguration;
+import com.liferay.analytics.settings.rest.manager.AnalyticsSettingsManager;
+import com.liferay.arquillian.extension.junit.bridge.junit.Arquillian;
+import com.liferay.portal.configuration.test.util.CompanyConfigurationTemporarySwapper;
+import com.liferay.portal.kernel.module.framework.ModuleServiceLifecycle;
+import com.liferay.portal.kernel.service.CompanyLocalService;
+import com.liferay.portal.kernel.service.UserLocalService;
+import com.liferay.portal.kernel.settings.SettingsFactoryUtil;
+import com.liferay.portal.kernel.test.ReflectionTestUtil;
+import com.liferay.portal.kernel.test.rule.AggregateTestRule;
+import com.liferay.portal.kernel.test.util.TestPropsValues;
+import com.liferay.portal.kernel.util.HashMapDictionaryBuilder;
+import com.liferay.portal.kernel.util.Http;
+import com.liferay.portal.kernel.util.Portal;
+import com.liferay.portal.test.rule.Inject;
+import com.liferay.portal.test.rule.LiferayIntegrationTestRule;
+import com.liferay.portal.test.rule.PermissionCheckerMethodTestRule;
+import com.liferay.segments.asah.connector.internal.constants.AnalyticsConnectionPropertiesConstants;
+import com.liferay.segments.service.SegmentsEntryLocalService;
+import com.liferay.segments.service.SegmentsEntryRelLocalService;
+
+import java.lang.reflect.Constructor;
+
+import java.util.Objects;
+
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.ClassRule;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+
+import org.osgi.framework.Bundle;
+import org.osgi.framework.BundleContext;
+import org.osgi.framework.FrameworkUtil;
+
+/**
+ * @author Mikel Lorza
+ */
+@RunWith(Arquillian.class)
+public class IndividualSegmentsCheckerTest {
+
+	@ClassRule
+	@Rule
+	public static final AggregateTestRule aggregateTestRule =
+		new AggregateTestRule(
+			new LiferayIntegrationTestRule(),
+			PermissionCheckerMethodTestRule.INSTANCE);
+
+	@Before
+	public void setUp() throws Exception {
+		_setIndividualSegmentsCheckerTest();
+	}
+
+	@Test(expected = Test.None.class)
+	public void testCheckIndividualSegments() throws Exception {
+		try (CompanyConfigurationTemporarySwapper
+				companyConfigurationTemporarySwapper =
+					new CompanyConfigurationTemporarySwapper(
+						TestPropsValues.getCompanyId(),
+						AnalyticsConfiguration.class.getName(),
+						HashMapDictionaryBuilder.<String, Object>put(
+							"liferayAnalyticsDataSourceId",
+							AnalyticsConnectionPropertiesConstants.
+								LIFERAY_ANALYTICS_DATA_SOURCE_ID
+						).put(
+							"liferayAnalyticsEnableAllGroupIds",
+							AnalyticsConnectionPropertiesConstants.
+								LIFERAY_ANALYTICS_ENABLE_ALL_GROUP_IDS
+						).put(
+							"liferayAnalyticsFaroBackendSecuritySignature",
+							AnalyticsConnectionPropertiesConstants.
```

Comment:
This is not a real value, so better is use random, otherwise readers will think that this is a real liferayAnalyticsFaroBackendSecuritySignature